Makefile

```diff
@@ -547,6 +550,15 @@ endif
 	$(verbose) echo "override MCS_CATS := $(MCS_CATS)" >> $(headerdir)/build.conf
 	$(verbose) $(INSTALL) -m 644 $(support)/Makefile.devel $(headerdir)/Makefile
 
+########################################
+#
+# Rebuild policy interface database
+#
+rebuild-interface-db:
+	$(verbose) rm -rf $(headerdir)
+	$(verbose) $(MAKE) install-headers
+	$(verbose) $(SEPOLGEN_IFGEN) $(VERBOSE_FLAG) -i $(headerdir)
```

Comment:
This should probably be reworked to have `build-interface-db` and `rebuild-interface-db` targets which would work like the `load` and `reload` targets in the `Rules.monolithic`.